test/lib.test.ts

```diff
@@ -16,7 +16,7 @@ import {
   publishedDefinitions,
   publishedSchemas,
   signIDXSchemas,
-} from '..'
+} from '../src'
```

Comment:
Can you revert this please? I prefer to have these integration tests run on the built package rather than the source to make sure there's no issue with the build itself.